src/OrchardCore.Modules/OrchardCore.Workflows/Http/Filters/WorkflowActionFilter.cs

```diff
@@ -39,45 +39,17 @@ IWorkflowInstanceStore workflowinstanceRepository
 
             if (workflowDefinitionEntries.Any())
             {
-                var workflowDefinitionIds = workflowDefinitionEntries.Select(x => int.Parse(x.WorkflowId)).ToList();
+                var workflowDefinitionIds = workflowDefinitionEntries.Select(x => Int32.Parse(x.WorkflowId)).ToList();
```

Comment:
I was under the impression that we decided to stick with the ASP.NET Core team's convention of using the C# alias instead of the .NET type when invoking members. Are you sure this change is correct? I don't mind either way, just that everyone is applying the same rule consistently. /cc @Jetski5822